lxd/db/instances.go

```diff
@@ -243,14 +243,14 @@ SELECT nodes.id, nodes.address
 	return address, nil
 }
 
-// GetInstanceNamesByNodeAddress returns the names of all containers grouped by
+// GetInstanceAndProjectNamesByNodeAddress returns the names of all containers grouped by
 // cluster node address.
 //
 // The node address of containers running on the local node is set to the empty
 // string, to distinguish it from remote nodes.
 //
 // Containers whose node is down are addeded to the special address "0.0.0.0".
-func (c *ClusterTx) GetInstanceNamesByNodeAddress(projects []string, filter InstanceFilter) (map[string][]string, error) {
+func (c *ClusterTx) GetInstanceAndProjectNamesByNodeAddress(projects []string, filter InstanceFilter) (map[string][][2]string, error) {
```

Comment:
This is going to need a comment update too to describe better what is being returned (otherwise it requires the user to read the function's code to figure out whats in 0 and 1 elements of the returned array).

Also whilst you're at it, can you remove references to "containers" in this function's comment and change it to "instances".